charts/gsp-cluster/requirements.lock

```diff
@@ -14,5 +14,5 @@ dependencies:
 - name: harbor
   repository: https://helm.goharbor.io/
   version: 1.1.0
-digest: sha256:a0a175e0f1540a2d32318a1bdb34c6b0963e7d0b5792ae478ea046ec189a8e0b
-generated: "2019-06-14T13:23:15.105558+01:00"
+digest: sha256:a991f9481b834b066828ab2df050ab705b51d7ae221df979bc4cc404a8344076
```

Comment:
Should we fear?

Reply:
oof I didn't notice that ... how can the digest change but the tarball not!?